gammapy/time/models.py

```diff
@@ -84,7 +84,7 @@ def phase(self, time):
         f1 = pars['f1'].value
         f2 = pars['f2'].value
 
-        t = time - time_0
+        t = (time - time_0)*86400.0
```

Comment:
Can you write `u.day.to(u.second)` here instead of the "magic constant" 86400?
This is assuming you have `from astropy import units as u` at the top of the file.